src/beachbot/robot/jetsonrobotv1.py

```
@@ -19,7 +20,7 @@ def __init__(self):
             usbcam = UsbCameraOpenCV()
             self.cameradevices[RobotInterface.CAMERATYPE.GRIPPER] = usbcam
         except Exception:
-            pass
+            logger.warning("Gripper camera not detected, RobotInterface.CAMERATYPE.GRIPPER not available!")
```

You should rethrow the exception or handle it. Not just a warning and pass.